src/content/self-hosted/on-premise.mdx

```diff
@@ -0,0 +1,195 @@
+
+```console
+$ kubectl create namespace okteto
+```
+
+### Configuring your Okteto instance
+
+Configure the Okteto helm values file with the following information:
+
+```yaml
+auth:
+  openid:
+    enabled: true
+    clientId: clientid
+    clientSecret: clientSecret
+    group: my-group
+    endpoints:
+      issuer: https://your-provider
+      authorization: https://your-provider/authorization
+    mapping:
+      externalIDKey: nickname
+      nameKey: name
+      emailKey: email
+      pictureKey: picture
+      groupsKey: groups
+
+email: admin@example.com
+license: 1234567890ABCD==
+subdomain: dev.example.com
+cluster:
+  endpoint: "https://your-kubernets-cluster-api-endpoint"
+
+ingress-nginx:
+  enabled: false
+
+registry:
+  storage:
+    filesystem:
+      enabled: true
+      persistence:
+        enabled: true
+        size: 400Gi
+    cloud:
+      enabled: false
+```
+
+And configure the field [ingress](/docs/self-hosted/administration/configuration/#ingress) based on the Ingress Controller running in your cluster.
+
+### Installing your Okteto instance
+
+Install the latest version of Okteto by running:
+
+```console
+helm install okteto okteto/okteto -f config.yaml --namespace=okteto
+```
+
+After a few seconds, all the resources will be created. The output will look something like this:
+
+```
+Release "okteto" has been installed. Happy Helming!
+NAME: okteto
+LAST DEPLOYED: Thu Mar 26 18:07:55 2020
+NAMESPACE: okteto
+STATUS: deployed
+REVISION: 1
+TEST SUITE: None
+NOTES:
+1. Create the following DNS record, pointing it to the NGINX controller service External-IP:
+- "*.dev.example.com"
+
+You can retrieve the External IP by running:
+    kubectl get service -l=app.kubernetes.io/name=ingress-nginx,app.kubernetes.io/component=controller --namespace=okteto
+
+2. Once you create both DNS entries you can access your Okteto instance at this URL:
+https://okteto.dev.example.com
+```
+
+> You can also install a specific version by including the `--version` argument.
+
+### Retrieve the Ingress Controller IP address
+
+We can use the instructions `kubectl` to fetch the address that has been dynamically allocated by AKS to the NGINX Ingress we've just installed and configured as a part of Okteto:
+
+```console
+kubectl get service -l=app.kubernetes.io/name=ingress-nginx,app.kubernetes.io/component=controller --namespace=okteto
+```
+
+The output will look something like this:
+
+```
+NAME                             TYPE           CLUSTER-IP   EXTERNAL-IP                          PORT(S)                                     AGE
+okteto-ingress-nginx-controller  LoadBalancer   10.0.7.73    a519c8b3b27f94...aks.microsoft.com   80:30795/TCP,443:32481/TCP,1234:30885/TCP   5m
+```
+
+You'll need to take the `EXTERNAL-IP` address and add it to your DNS for the domain you have chosen to use. In AzureDNS, this is done by creating an `A` record with the name  `*`, pointing to the IP of the Load Balancer.
```

Comment:
What about Buildkit?

Reply:
In this guide, it's exposed as an ingress